core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -45,6 +45,34 @@ oppia.factory('explorationData', [
     // Put exploration variables here.
     var explorationData = {
       explorationId: explorationId,
+      autosaveChangeList: function(changeList, successCallback, errorCallback) {
+        var draftAutosaveUrl = '/createhandler/autosave_draft/' + explorationId;
```

Comment:
I'm not sure if I get you here. Are you suggesting that I remove successCallback and errorCallback arguments and instead replace them by whatever I'm passing while calling this method ? 
If that's what you are suggesting, then I think I will run into circular dependency errors, since `changeList` will have to be injected into this factory!

If otherwise, please elaborate.